src/dev/build/tasks/os_packages/docker_generator/run.ts

```diff
@@ -37,19 +37,25 @@ export async function runDockerGenerator(
   config: Config,
   log: ToolingLog,
   build: Build,
-  ubi: boolean = false
+  flags: {
+    architecture: string;
+    context: boolean;
+    image: boolean;
+    ubi: boolean;
+  }
 ) {
   // UBI var config
-  const baseOSImage = ubi ? 'docker.elastic.co/ubi8/ubi-minimal:latest' : 'centos:8';
+  const baseOSImage = flags.ubi ? 'docker.elastic.co/ubi8/ubi-minimal:latest' : 'centos:8';
   const ubiVersionTag = 'ubi8';
-  const ubiImageFlavor = ubi ? `-${ubiVersionTag}` : '';
+  const ubiImageFlavor = flags.ubi ? `-${ubiVersionTag}` : '';
 
   // General docker var config
   const license = build.isOss() ? 'ASL 2.0' : 'Elastic License';
   const imageFlavor = build.isOss() ? '-oss' : '';
   const imageTag = 'docker.elastic.co/kibana/kibana';
   const version = config.getBuildVersion();
-  const artifactTarball = `kibana${imageFlavor}-${version}-linux-x86_64.tar.gz`;
+  const artifactArchitecture = flags.architecture === 'x64' ? 'x86_64' : 'aarch64';
```

Comment:
Can we match on aarch to follow same order/pattern here? https://github.com/elastic/kibana/pull/84831/files#diff-a7e161332f3a6fb3494f1866ecc9e560febfdf913e7db4c8627d9aac623ecc1cR32

Reply:
Pushed with 749c203